Keas.Mvc/ClientApp/components/People/PeopleContainer.tsx

```diff
@@ -141,6 +150,66 @@ export default class PeopleContainer extends React.Component<{}, IState> {
     return filters.every(f => person.tags.includes(f));
   }
 
+  private _createPerson = async (
+    person: IPerson,
+  ) => {
+    const index = this.state.people.findIndex(x => x.person.userId === person.userId);
+    if (index !== -1)
+    {
+      // if we somehow already have this person, return
+      return;
+    }
+    person.teamId = this.context.team.id;
+    person = await this.context.fetch(`/api/${this.context.team.name}/people/create`, {
+      body: JSON.stringify(person),
+      method: "POST"
+      })
+      .catch(err => {
```

Comment:
@laholstege With async/await instead of using .catch() you should just wrap a try/catch statement.   .catch() is for when you are using the old promise.then().catch() style.